tests/models/recurrent_gemma/test_modeling_recurrent_gemma.py

```diff
@@ -413,6 +413,10 @@ def _check_hidden_states_for_generate(
     def test_initialization(self):
         pass
 
+    @unittest.skip(reason="TODO @gante not super important and failing.")
+    def test_inputs_embeds_matches_input_ids_with_generate(self):
+        pass
```

Comment:
Failing on `main` too.

I suspect this didn't get detected in the (recent) PR that added it because of our test fetched, which only checks the main models on these wide changes (?)

Reply:
https://github.com/huggingface/transformers/issues/32913

(The issue catches a wider net than skipping `test_inputs_embeds_matches_input_ids_with_generate`, as this skip is merely a symptom of the root problem :) )